romana-install/roles/provision/create-aws-hosts/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
   command: >
     aws ec2 describe-instances --output json
     --filters 'Name=tag:aws:cloudformation:stack-name,Values={{ stack_name }}' 'Name=instance-state-name,Values=running'
-    --query 'Reservations[*].Instances[0].{Name:Tags[?Key==`Name`].Value|[0],IP:PublicIpAddress,Host:Tags[?Key==`StackHost`].Value|[0],Role:Tags[?Key==`StackRole`].Value|[0]}'
+    --query 'Reservations[*].Instances[0].{Name:Tags[?Key==`Name`].Value|[0],IP:PublicIpAddress,Role:Tags[?Key==`StackRole`].Value|[0]}'
```

Comment:
Before, I was maintaining a "stack_host" variable on each host in the inventory and using it during the config stage. It'd be the key into more-specific data about the host/type (pre-calculated IPs and so on).

With the other changes to the config stage, this variable stopped being necessary so it's removed from the CloudFormation template and from the cloudformation query (the specific line above)